warehouse/query-core/src/test/java/datawave/query/util/WiseGuysIngest.java

```diff
@@ -597,116 +613,130 @@ public static void writeItAll(AccumuloClient client, WhatKindaRange range) throw
             mutation.put("fi\u0000" + "UUID", lcNoDiacriticsType.normalize("SOPRANO") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
-                            timeStamp, emptyValue);
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
             // names
             mutation.put("fi\u0000" + "NAME", lcNoDiacriticsType.normalize("ANTHONY") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
-                            timeStamp, emptyValue);
-            mutation.put("fi\u0000" + "NAME", lcNoDiacriticsType.normalize("MEADOW") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility, timeStamp,
-                            emptyValue);
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
+            mutation.put("fi\u0000" + "NAME", lcNoDiacriticsType.normalize("MEADOW") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
             // genders
-            mutation.put("fi\u0000" + "GENDER", lcNoDiacriticsType.normalize("MALE") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility, timeStamp,
-                            emptyValue);
+            mutation.put("fi\u0000" + "GENDER", lcNoDiacriticsType.normalize("MALE") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
             mutation.put("fi\u0000" + "GENDER", lcNoDiacriticsType.normalize("FEMALE") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
-                            timeStamp, emptyValue);
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
             // ages
-            mutation.put("fi\u0000" + "AGE", numberType.normalize("16") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility, timeStamp, emptyValue);
-            mutation.put("fi\u0000" + "AGE", numberType.normalize("18") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility, timeStamp, emptyValue);
+            mutation.put("fi\u0000" + "AGE", numberType.normalize("16") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
+            mutation.put("fi\u0000" + "AGE", numberType.normalize("18") + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility,
+                            timeStamp + sopranoTimeStampDelta, emptyValue);
 
             // geo
             for (String normalized : ((OneToManyNormalizerType<Geometry>) geoType).normalizeToMany("POINT(20 20)")) {
-                mutation.put("fi\u0000" + "GEO", normalized + "\u0000" + datatype + "\u0000" + corleoneUID, columnVisibility, timeStamp, emptyValue);
+                mutation.put("fi\u0000" + "GEO", normalized + "\u0000" + datatype + "\u0000" + sopranoUID, columnVisibility, timeStamp + sopranoTimeStampDelta,
```

Comment:
Most of these Keys only changed timestamps and UIDs did not change.  This one's UID and timestamp went form corleone to soprano.  I'm wondering if that was a mistake.

Reply:
Disregard.  I think it's meant to align with the prior keys.  I see a similar change below.